src/inference/tests/functional/ov_extension_test.hpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include "common_test_utils/test_common.hpp"
+#include "openvino/runtime/core.hpp"
+
+class OVExtensionTests : public CommonTestUtils::TestsCommon {
+public:
+    ov::Core core;
+
+    void test() {
+        std::string model = R"V0G0N(
```

Comment:
old version of IR. IRv11?

Reply:
Yes. I missed it. It will be addressed in the next PR